content/v1.14/concepts/_index.md

```diff
@@ -4,53 +4,70 @@ weight: 100
-operator facing, but may optionally offer an application developer facing
-composite resource claim that acts as a proxy for a composite resource. You can
-learn more about all of these concepts in the [composition documentation].
-
-<!-- Named Links -->
-
-[Packages]: {{<ref "packages" >}}
-[CRDs]: https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/
-[controllers]: https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#custom-controllers
-[providers documentation]: {{<ref "providers" >}}
-[Upbound Marketplace]: https://marketplace.upbound.io
-[managed resources documentation]: {{<ref "managed-resources" >}}
-[composition documentation]: {{<ref "./compositions" >}}
+Crossplane extends Kubernetes allowing Kubernetes to create and manage
+resources external to Kubernetes. Crossplane enables platform engineers to
+create custom APIs and abstractions combining both native Kubernetes
+resources and cloud resources under a single control plane.
+
+Crossplane uses multiple core components to manage the various elements of
+building and managing external resources through Kubernetes. 
+
+* [**The Crossplane pods**]({{<ref "./pods">}}) include the core Crossplane pod and
+  Crossplane RBAC manager pod. Together these pods manage all Crossplane
+  components and resources. 
+
+* [**Providers**]({{<ref "./providers">}}) connect Kubernetes to any external
+  provider, like AWS, Azure or GCP. Providers translate Kubernetes native
+  manifests and API calls into external API calls. Providers are responsible for
+  creating, deleting and managing the lifecycle of their resources.
+
+* [**Managed resources**]({{<ref "./managed-resources">}}) are Kubernetes objects
+  representing things the Provider created outside of Kubernetes. Creating a
+  managed resource in Kubernetes requires a Provider to create a resource.
+  Deleting a managed resource requires a Provider to delete the associated
+  external resource.
+
+* [**Compositions**]({{<ref "./composite-resources">}}) are a template of managed
+  resources. Compositions describe more complex deployments, combining multiple
+  managed resources and any resource customizations, like the size of a database
+  or the cloud provider region.
+
+* [**Composite Resource Definitions**]({{<ref "./composite-resource-definitions">}})
+  represent a custom API, created by platform engineers and consumed by
+  developers or end users. Composite resource definitions use an OpenAPIv3
+  schema to further extend Kubernetes with custom API endpoints, revisions and
+  more. 
+
+* [**Composite Resources**]({{<ref "./composite-resources">}}) represent all the
+  objects created by a user calling the custom API. Every time a user access the
+  custom API Crossplane creates a single Composite Resource and links all
+  the related managed resources to it. 
+
+* [**Claims**]({{<ref "./claims">}}) are like Composite Resources, but exist
```

Comment:
mention that the point of this is for developer/team access?

Reply:
Added